worlds/alttp/__init__.py

```diff
@@ -805,12 +805,15 @@ def get_filler_item_name(self) -> str:
         return GetBeemizerItem(self.multiworld, self.player, item)
 
     def get_pre_fill_items(self):
-        res = []
+        res = [self.create_item(name) for name in ('Red Pendant', 'Blue Pendant', 'Green Pendant', 'Crystal 1',
```

Comment:
This looks wrong to me, why are they suddenly needed to be added to state?

Reply:
Plando runs before pre_fill, before they would be added to the state normally. Without this change, players cannot plando items into locations locked behind rewards (including most notably, Ganon's Tower).